docs/howto/how-to-use-nnfw-python-api.md

```diff
@@ -0,0 +1,77 @@
+# How to Use NNFW PYTHON API
+
+## Prepare nnpackage
+
+### Convert tensorflow pb file to nnpackage
+
+Follow the [compiler guide](https://github.com/Samsung/ONE/blob/master/docs/nncc/v1.0.0/tutorial.md) to generate nnpackge from tensorflow pb file
+
+### Convert tflite file to nnpackage
+
+Please see [model2nnpkg](https://github.com/Samsung/ONE/tree/master/tools/nnpackage_tool/model2nnpkg) for converting from tflite model file.
+
```

Comment:
I understood that you are asking me to provide a guide to use a model example that is already converted to nnpackage, rather than providing a guide to convert nnpackage, is that correct?

Reply:
Right. It is very kind to explain how to prepare nnpackage, but this document can concentrate on how to use python api by guiding the existing nnpackage in this repo.